packages/fxa-payments-server/server/lib/server.js

```diff
@@ -30,12 +30,6 @@ module.exports = () => {
   const { cors, routing } = require('fxa-shared/express').express();
   const { v4: uuid } = require('uuid');
 
-  const {
-    tagCriticalEvent,
-    buildSentryConfig,
-    tagFxaName,
-  } = require('fxa-shared/sentry');
```

Comment:
This change seems unrelated to the PR title

Reply:
I was getting a bunch of lint errors and warnings during testing that were clogging my console. This was one of them. Those items are declared but never used. I can revert in this PR. I just did it for cleanup. Please let me know!

Reply:
Interesting that these aren't failing in main. Are you running your lint against just the code you changed, or against the whole project?

Reply:
This was a while back so I can't quite remember. Considering it's payments-server, I'm guessing it was the whole project or just that package. 🤔